pkg/commands/workload.go

```diff
@@ -390,6 +402,40 @@ func (opts *WorkloadOptions) checkToPublishLocalSource(taggedImage string, c *cl
 	return okToPush
 }
 
+func (opts *WorkloadOptions) loadExcludedPaths(c *cli.Config) []string {
+	exclude := []string{}
+	if opts.ExcludePathFile != "" {
+		p := filepath.Join(opts.LocalPath, opts.ExcludePathFile)
+		if _, err := os.Stat(p); errors.Is(err, os.ErrNotExist) {
+			return exclude
+		}
+
+		f, err := os.Open(p)
+		if err != nil {
+			c.Infof("Unable to read %s file.\n", opts.ExcludePathFile)
```

Comment:
is it possible to add a test for this? (have a file with settings that it cant open with chmod set for root user possibly)